A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/panel/panel.cfg

```diff
@@ -124,6 +124,13 @@ texture=$MFD2
 
 htmlgauge00=Airliners/A320_Neo/MFD/A320_Neo_MFD.html?Index=2, 0,0,1280,1280
 
+[VCockpit17]
+size_mm=1280,1280
```

Comment:
size is wrong

Reply:
It does not display anything

Reply:
We should at least have it displayed right so when we do try to get it to display something we have to change less